sdk/keyvault/azure-keyvault-administration/azure/keyvault/administration/_internal/http_challenge.py

```diff
@@ -10,7 +10,9 @@
 
 
 class HttpChallenge(object):
-    def __init__(self, request_uri: str, challenge: str, response_headers: "MutableMapping[str, str]" = None) -> None:
+    def __init__(
+        self, request_uri: str, challenge: str, response_headers: "Optional[MutableMapping[str, str]]" = None
```

Comment:
Should type hint be quoted?

Reply:
In this case it needs to be since we're only importing with `if TYPE_CHECKING:`